src/js/page_sections/Header.js

```diff
@@ -59,7 +59,7 @@ class Header extends Component {
               </div>
             </div>
             <div className="col-xs-6 coa-Header__right-controls">
-              <div className="d-none d-md-block">
+              <div className="d-none d-lg-block">
```

Comment:
These links show in the Sketch mocks on medium screens. If we are going to change them, let's make sure that is @cthibaultatx's intent to stray from what was designed.

Reply:
📟